apps/package.json

```diff
@@ -326,6 +326,7 @@
     "react-router": "3.2.6",
     "react-router-dom": "^6.26.0",
     "react-select": "^1.2.1",
+    "react-select-5": "npm:react-select@5",
```

Comment:
What's the difference between `react-select-5` and `react-select`?

Reply:
We have a really old version from ~7 years ago but I wanted this code to use a new version along with its TypeScript support.  The old one is used in a variety of places in our product and the upgrade process looks fairly involved, so I figured using the newer version here would be the least risky path forward.